DESCRIPTION

```diff
@@ -27,7 +27,7 @@ License: MIT + file LICENSE
 URL: https://rstudio.github.io/bslib/, https://github.com/rstudio/bslib
 BugReports: https://github.com/rstudio/bslib/issues
 Depends:
-    R (>= 2.10)
+    R (>= 3.5)
```

Comment:
This is forces every shiny app to require 3.5, which will cause pain for at least a few people. Is this just for `isFALSE`?